test/tools/RegressionChecker/src/main.js

```diff
@@ -85,6 +85,20 @@ if (jsonTypeCheck(RCjson, "webnn", "boolean")) {
     webnn = RCjson.webnn;
 }
 
+var getLDLibraryPath = function() {
+    let basePath = chromiumPath.slice(0, -14) + "Versions/";
```

Comment:
@cuiyanx  Here's magic number '14' , which is the string of length as 14?

Reply:
The path of chromium is **'.../chromiumBuild/f456c6d/mac_x64_SUCCEED/chromium-mac/Chromium.app/Contents/MacOS/Chromium'**. 
And the path of LD library is **'.../chromiumBuild/f456c6d/mac_x64_SUCCEED/chromium-mac/Chromium.app/Contents/Versions/75.0.3739.0/Chromium Framework.framework/Libraries/'**.
To get the path of LD library from the path of chromium, we need to tailor string of chromium path. And the length of 'MacOS/Chromium' is 14 that must be deleted.

Reply:
OK, It's better to add this explanation above this code.